articles/azure/azs-how-create-availability-set.md

```diff
@@ -0,0 +1,166 @@
+
+## Creating an availability set via the portal
+
+1. The first step when initially creating an availability set is to create the resource group for the availability set.
+
+    ![Microsoft Azure availability set step 1](images/azs-portal-create-availability-set1.png)
+
+2. Name the resource group.
+
+    ![Microsoft Azure availability set step 2](images/azs-portal-create-availability-set2.png)
+
+3. Once the resource group has been created if you navigate into it and select **Overview** then you will be able to select **Create resources**.
+
+    ![Microsoft Azure availability set step 3](images/azs-portal-create-availability-set3.png)
+
+4. This will load the marketplace, if you search for **Availability Set**, you will be given the option to create one.
+
+    ![Microsoft Azure availability set step 4](images/azs-portal-create-availability-set4.png)
+
+5. This will bring up the **Create availability** set blade. Here you can set the name of the set and how many fault and update domains you want it to have. The resource group should be the same as the one you created earlier. As for the option for managed disks, select the option for which matches the VMs disks you will be adding to the set. Once done select **Create** and the availability set will be deployed.
+
+    ![Microsoft Azure availability set step 5](images/azs-portal-create-availability-set5.png)
+
+6. Now when you are creating the VM's you want to be a part of the availability set, when you get to step 3, under **High Availability** select the availability set you want to use. Make sure to set the disk type of the VM first.
+
+    ![Microsoft Azure availability set step 6](images/azs-portal-create-availability-set6.png)
+
+7. Once you have created the VM's if you navigate to the availability set, in overview you will be able to see your VM's and you will be able to see what domains they are located on.
+
+    ![Microsoft Azure availability set step 7](images/azs-portal-create-availability-set7.png)
+
+## Creating an availability set via PowerShell
+
+The following PowerShell code example will create an empty resource group and then deploy an availability set to it.
+
+```PowerShell
+$Resourcegroup = "myResourceGroupAvailability"
+$Availabilityset = "myAvailabilitySet"
+
+New-AzureRmResourceGroup -Name $Resourcegroup -Location 'frn00006'
+
+New-AzureRmAvailabilitySet `
+   -Location "frn00006" `
+   -Name $Availabilityset `
+   -ResourceGroupName $Resourcegroup `
+   -Sku classic `
+   -PlatformFaultDomainCount 2 `
+   -PlatformUpdateDomainCount 2  
+```
+
+The following code is a variation of the script required to create a VM. This code will add the VM to the availability set that was created in the code outlined above.
```

Comment:
The following PowerShell code sample is a variation of the script required to create a VM. This code creates the VM and then adds it to the availability set that you created with the previous script.

Reply:
done